### ISSUE TITLE
Warning is issued if column value contains : (Colon)

### ISSUE THREAD

**Issue by trks1970** [database-rider/database-rider#335]
I have a dataset, where several text fields contain values like:
"ABC: this is abc"
"DEF: this is def"
"GHA: this is gha"

When importing with @DataSet warnings are issued for all values before the colon, eg:
Could not find script engine by name 'ABC'
Could not find script engine by name 'DEF'
Could not find script engine by name 'GHA'

**Comment by rmpestano** [comment]
Hi @trks1970, that warning is relatos to [scriptable datasets](https://github.com/database-rider/database-rider#311-scriptable-datasets), the colon is used to identify the script engine.

**Comment by rmpestano** [comment]
I think we should have a list of well known script engines instead trying to guess the engine whenever we find a colon in the dataset

**Comment by rmpestano** [comment]
Can you try with `1.25.1-SNAPSHOT`?  The warn should be gone. To enable snapshot you have to config snapshots repo on your build: 

```<repositories>
    <repository>
        <snapshots/>
        <id>snapshots</id>
        <name>libs-snapshot</name>
        <url>https://oss.sonatype.org/content/repositories/snapshots</url>
    </repository>
</repositories>```

Thanks for the feedback

**Comment by rmpestano** [comment]
This fix is available in v`1.26.0`
